src/libexec/zmfixperms

```diff
@@ -627,4 +627,22 @@ for i in slapd.conf slapd.conf.default slapd.ldif slapd.ldif.default; do
   fi
 done
 
+##### Fix permissions for ldap and proxy #####
+
+if [ -x /opt/zextras/common/sbin/nginx ]; then
+  chown ${root_user}:${zextras_group} /opt/zextras/common/sbin/nginx
+  chmod 750 /opt/zextras/common/sbin/nginx
+
+  # changing permission will reset capabilities so set it again
+  setcap CAP_NET_BIND_SERVICE=+ep /opt/zextras/common/sbin/nginx
+fi
+
+if [ -x /opt/zextras/common/libexec/slapd ]; then
+  chown ${root_user}:${zextras_group} /opt/zextras/common/libexec/slapd
```

Comment:
Changes make sense to me, I just wonder if there is need to give permission to root user at all

Reply:
I think that should work also with zextras:zextras instead of root:zextras, we can refine after